samples/azure-quantum/shorter-quantum-circuits/shorter-quantum-circuits-dataset.ipynb

```diff
@@ -0,0 +1,782 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Numerical results for the [Shorter Quantum Circuits](https://arxiv.org/abs/2203.10064) paper "
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Datasets overview"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### List of datasets \n",
+    "\n",
+    "We provide numerical results for approximating diagonal rotation gates $\\exp(i\\phi Z)$ for the Clifford+$T$ \n",
+    "and Clifford+$\\sqrt{T}$ gate sets. The corresponding dataset files begin with the prefixes `clifford-t-` and `clifford-root-t-`.\n",
+    "For each gate set, we have two sets of target rotation angles: Fourier angles ($\\phi = \\frac{\\pi}{2^n}$) and random angles.\n",
+    "The corresponding dataset file names end in `fourier-angles.tar.gz` and `random-angles.tar.gz`.\n",
+    "For each target angle, we construct a series of approximations with increasing diamond norm accuracies using the following protocols:\n",
+    "\n",
+    "* diagonal approximation \n",
+    "* fallback approximation\n",
+    "* mixed diagonal approximation \n",
+    "* mixed fallback approximation\n",
+    "\n",
+    "For a summary of the numerical results, refer to [Table 1](https://arxiv.org/pdf/2203.10064v1.pdf#page=6) in the paper. \n",
+    "The dataset naming convention is `<gate-set>-<angles-kind>.tar.gz`"
```

Comment:
Could you add some sentence that describes that the following code cell downloads the data set files to be used in the notebook.

Reply:
Added